app/src/main/java/com/cvtracker/vmd/custom/SortSwitchView.kt

```diff
@@ -23,6 +24,7 @@ class SortSwitchView @JvmOverloads constructor(context: Context, attrs: Attribut
             val view = LayoutInflater.from(context)
                 .inflate(R.layout.view_sort_switch_item, container, false).apply {
                     setOnClickListener {
+                        PrefHelper.primarySort = filter
```

Comment:
Why not moving this line to the presenter ? `onSortChanged`

Reply:
Done